core/src/test/java/apoc/refactor/CloneSubgraphTest.java

```diff
@@ -407,67 +403,61 @@ public void testCloneSubgraph_With_No_Nodes_But_With_Rels_And_Standins_Should_Do
+                    (row) -> {}
+            )
         );
     }
 
     @Test
     public void testCloneSubgraph_With_A_Null_Element_In_Standin_Pair_Should_Throw_Exception()  {
-        exceptionGrabber.expectMessage(STANDIN_SYNTAX_EXCEPTION_MSG);
-
-        TestUtil.testCall(db,
-                "MATCH (root:Root{name:'A'})-[*]-(node) " +
-                        "WITH root, collect(DISTINCT node) as nodes " +
-                        "CALL apoc.refactor.cloneSubgraph(nodes, [], {standinNodes:[[root, null]]}) YIELD input, output, error " +
-                        "WITH collect(output) as clones, collect(output.name) as cloneNames " +
-                        "RETURN cloneNames, size(cloneNames) as cloneCount, none(clone in clones WHERE (clone)--()) as noRelationshipsOnClones, " +
-                        " single(clone in clones WHERE clone.name = 'node5' AND clone:Oddball) as oddballNode5Exists, " +
-                        " size([clone in clones WHERE clone:Node]) as nodesWithNodeLabel",
-                (row) -> {
-                    assertThat((List<String>) row.get("cloneNames"), containsInAnyOrder("node1", "node2", "node3", "node4", "node5", "node8", "node9", "node10", "node6", "node7"));
-                    assertThat(row.get("cloneCount"), is(10L));
-                    assertThat(row.get("noRelationshipsOnClones"), is(true));
-                    assertThat(row.get("oddballNode5Exists"), is(true));
-                    assertThat(row.get("nodesWithNodeLabel"), is(10L));
-                }
+        assertThrows(
+            STANDIN_SYNTAX_EXCEPTION_MSG,
+            QueryExecutionException.class,
+            () -> TestUtil.testCall(db,
+                    "MATCH (root:Root{name:'A'})-[*]-(node) " +
+                            "WITH root, collect(DISTINCT node) as nodes " +
+                            "CALL apoc.refactor.cloneSubgraph(nodes, [], {standinNodes:[[root, null]]}) YIELD input, output, error " +
+                            "WITH collect(output) as clones, collect(output.name) as cloneNames " +
+                            "RETURN cloneNames, size(cloneNames) as cloneCount, none(clone in clones WHERE (clone)--()) as noRelationshipsOnClones, " +
+                            " single(clone in clones WHERE clone.name = 'node5' AND clone:Oddball) as oddballNode5Exists, " +
+                            " size([clone in clones WHERE clone:Node]) as nodesWithNodeLabel",
+                    (row) -> {
+                        assertThat((List<String>) row.get("cloneNames"), containsInAnyOrder("node1", "node2", "node3", "node4", "node5", "node8", "node9", "node10", "node6", "node7"));
+                        assertThat(row.get("cloneCount"), is(10L));
+                        assertThat(row.get("noRelationshipsOnClones"), is(true));
+                        assertThat(row.get("oddballNode5Exists"), is(true));
+                        assertThat(row.get("nodesWithNodeLabel"), is(10L));
+                    }
```

Comment:
```suggestion
                    (row) -> {}
```

In case this one also throws, can not these assertions be removed from this test as well?

Reply:
Yep!